appspec.yml.deploy

```diff
@@ -8,13 +8,13 @@ hooks:
     - location: deploy_scripts/stop_cron.sh
       timeout: 30
   AfterInstall:
-    - location: deploy_scripts/install_dependencies.sh
+    - location: deploy_scripts/install_dependencies_django.sh
       timeout: 300
-    - location: deploy_scripts/run_management_commands.sh
+    - location: deploy_scripts/run_management_commands_django.sh
       timeout: 300
   ApplicationStart:
-    - location: deploy_scripts/application_start.sh
+    - location: deploy_scripts/restart_httpd.sh
       timeout: 20
   ValidateService:
-    - location: deploy_scripts/validate_service.sh
+    - location: deploy_scripts/curl_index.sh
```

Comment:
I think you may be able to curl the status endpoint, which is a slightly stronger test?

Reply:
Updated this in the last commit - @helrond just want to confirm that the change I made is what you're talking about before merging to base.

Reply:
That's right @HaSistrunk